doc/nvim-tree-lua.txt

```diff
@@ -1615,6 +1615,7 @@ as per |:highlight|
 Default linked group follows name.
 
 NvimTreeSymlink
```

Comment:
We now have some inconsistency, as this group applies specifically to files.

We could add `NvimTreeFileSymlink` and deprecate `NvimTreeSymlink` however I don't think it is worth the effort of migration. It should be clear enough to the user as `NvimTreeFolderSymlink` exists.